CRM/Core/Resources/Common.php

```diff
@@ -189,6 +189,8 @@ protected static function coreResourceList($region) {
       "packages/jquery/plugins/jquery.mousewheel.min.js",
       "bower_components/select2/select2.min.js",
       "bower_components/select2/select2.min.css",
+      "bower_components/select2-4.x/dist/js/select2.min.js",
```

Comment:
@colemanw needs to be `select4.min.js`

Reply:
No I didn't rename the js files, it wasn't necessary.